src/EnergyPlus/WeatherManager.cc

```diff
@@ -7233,7 +7231,6 @@ namespace Weather {
         Real64 tz; // resultant tz meridian
         for (int i = -12; i <= 12; ++i) {
             if (temp > longl(i) && temp <= longh(i)) {
-                tz = i;
                 tz = mod(i, 24.0);
```

Comment:
[src/EnergyPlus/WeatherManager.cc:7237]:(style),[redundantAssignment],Variable 'tz' is reassigned a value before the old one has been used.